cmd/beaker/experiment.go

```diff
@@ -104,16 +104,13 @@ func newExperimentExecutionsCommand() *cobra.Command {
 		Short: "List the executions in an experiment",
```

Comment:
```suggestion
		Short: "List the jobs in an experiment",
```

It may be worth doing a quick pass to find other instances of `exec(ution)?` throughout and decide whether they should change to `job`.

Reply:
In this case, I think it's OK to leave this as execution since sessions don't belong to an experiment.

We have a lot of commands for listing executions:
 - `beaker experiment executions`
 - `beaker node executions`
 - `beaker group executions`
 - `beaker cluster executions`

Maybe we should just merge all of these into the `beaker job list` command with optional flags. What do you think?

Reply:
> Maybe we should just merge all of these into the beaker job list command with optional flags. What do you think?

This is my preference.  I think the term "Job" is a lot more intuitive to users.  Despite consistent treatment execution/task/session, when people talk about these they tend to say "interactive shell" or "batch job".

We should keep the `beaker session` command group, at least for now, but it seems like "execution" hasn't stuck and we should start phasing that out in favor of "job".

Reply:
I replaced these commands with flags on `beaker job list`. I added the `--experiment` flag which accepts multiple experiments. I didn't add a replacement for the `beaker group executions` command but this can be retrieved using `beaker groups experiments` combined with `beaker job list --experiment`.

I also replaced the `--all` flag with `--node all`. This is an admin-only API so I think it's better to make it a hidden feature than to expose a flag that will fail for most users. I think it's also a lot clearer what this option does now.